pkg/apis/pxc/v1/pxc_prestore_types.go

```diff
@@ -69,6 +69,8 @@ const (
 	RestoreSucceeded    BcpRestoreStates = "Succeeded"
 )
 
+const AnnotationUnsafePITR = "percona.com/unsafe-pitr"
```

Comment:
@spron-in please confirm this annotation

Reply:
looks good to me